charts/aws-vpc-cni/values.yaml

```diff
@@ -46,6 +46,7 @@ nodeAgent:
   networkPolicyAgentLogFileLocation: "/var/log/aws-routed-eni/network-policy-agent.log"
   enableIpv6: "false"
   metricsBindAddr: "8162"
+  metricsBindPort: "8162"
```

Comment:
This looks redundant `metricsBindPort` as we seem to be using `metricsBindAddr`.